### PR TITLE
DAOS-623 build: Make scons quiet quiet.

### PR DESCRIPTION
**Pull request by ashleypittman**
Update build so there's less output of quiet is used.

Required-githooks: true

Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

**Review by jolivier23 — APPROVED** [review]

**Comment by ashleypittman** [comment]
@brianjmurrell @phender can you review this please?

File: site_scons/env_modules.py [hunk 1]
```diff
@@ -56,7 +57,8 @@ def _module_func(self, command, *arguments):  # pylint: disable=no-self-use
 
         # pylint: disable=consider-using-with
         try:
-            print(f"Going to run {cmd}")
+            if not self._silent:
+                print(' '.join(cmd))
```
Comment (daltonbohning) [inline site_scons/env_modules.py:61]: Maybe define a `self.print` so we don't need this check everywhere?

File: src/tests/suite/SConscript [hunk 2]
```diff
@@ -97,7 +29,7 @@ def scons():
                                c_files + daos_test_tgt,
                                LIBS=['daos_common'] + libraries)
 
-    newenv = configure_cmocka(denv.Clone())
```
Comment (daltonbohning) [inline src/tests/suite/SConscript:100]: Why isn't this needed anymore?

File: site_scons/env_modules.py (hunk 1 above)
Comment (ashleypittman) [reply site_scons/env_modules.py:61]: I considered it but there are only two calls.

File: src/tests/suite/SConscript (hunk 2 above)
Comment (ashleypittman) [reply src/tests/suite/SConscript:100]: I tested - none of the distros we have in CI use a version of cmocka with the bug we're testing for.

**Review by phender — APPROVED** [review]
It would be nice if we could define `def _print(msg):` in one common file and import it in the other ones, but it does not look we already have a common shared daos scons python file.

**Comment by ashleypittman** [comment]
> It would be nice if we could define `def _print(msg):` in one common file and import it in the other ones, but it does not look we already have something like that set up already that would be easy to leverage.

I looked at this again after merging, we can add methods to envs but that doesn't help as much as you think it would as most of these prints don't have a env object available when the print is done.

**Pull request merged**